lib/ui/painting.dart

```diff
@@ -3910,8 +3910,11 @@ class FragmentProgram extends NativeFieldWrapperClass1 {
   static Future<FragmentProgram> compile({
     required ByteBuffer spirv,
     bool debugPrint = false,
-  }) {
-    return Future<FragmentProgram>(() => FragmentProgram._(spirv: spirv, debugPrint: debugPrint));
+  }) async {
+    // Delay compilation without creating a timer, which interacts poorly with the
+    // flutter test framework. See: https://github.com/flutter/flutter/issues/104084
+    await null;
```

Comment:
Is this different from `Future.microtask(FragmentProgram._(...))`?

Reply:
Ahh yeah, its basically the same thing.